### DISCUSSION TITLE
Where can I find the 'real' Spinner component?

### DISCUSSION THREAD

**Discussion by pldvd** [epam/UUI#1315] [Q&A] [answered]
According to the docs, there are 3 UI components for indicating loading state: Spinner, Waver and ProgressBar. But when importing the Spinner component from @epam/uui-promo, I actually get a Waver (not spinning, wavy thing). When searching the source code, and looking at the Spinner component, I understand that it exports what the docs call a 'Waver'... Can someone please point me in the right direction? Btw. I'm using the following version of the library: 4.10.1-beta.1.

**Comment by AlekseyManetov** [comment]
Сould you please share the docs where was mentioned Waver component?
You can find our documentation for spinner here - https://uui.epam.com/documents?id=spinner&mode=doc&skin=UUI4_promo&category=components
And for progress bars here - https://uui.epam.com/documents?id=progressBar&mode=doc&skin=UUI4_promo&category=components

**Comment by pldvd** [comment]
[This](https://uui.epam.com/documents?id=loaders&mode=doc&skin=UUI4_promo&category=guidelines) is where the Waver component is mentioned. As for the links you sent, please note that they are restriceted, I have no access to them.

**Reply by AlekseyManetov** [reply, marked as answer]
Sorry, I've fixed links in my previous message. The doc which you send related to the design library, in development we have a little bit another naming, because some of components was developed earlier than design library was created. If you need 'Spinner' in design terms, you can just use an animated icon from our @epam/assets package:
`import { ReactComponent as myIcon } from '@epam/assets/icons/loaders/circle_loader_30.svg';`

All icons previews could be find here - https://uui.epam.com/documents?id=icons&mode=doc&skin=UUI4_promo&category=assets

**Reply by pldvd** [reply]
Thank you, this will work I think!
